resources/migrate-to-swift.md

```diff
@@ -47,10 +47,34 @@ If you are using Cocoapods to manage your Adobe Experience Platform Mobile SDK d
   pod 'AEPUserProfile'
 ```
 
-Save the `Podfile` and run `pod repo update` to update your local Cocoapods repository.
+Save the `Podfile` and run `pod repo update` to update your local CocoaPods repository.
 
 Once the previous command is complete, run `pod install` or `pod update` to update the application dependencies.
 
+### SPM
+
+You can now also use SPM to manage your Adobe Experience Platform Mobile SDK dependencies. To add the AEP Packages to your application, from the Xcode menu select:
```

Comment:
Since this would be the first mention, I'd go with "You can now also use Swift Package Manger (SPM) to manage...".

Additionally, it should be "To add the Platform Packages to your application...", since using AEP is against branding guidelines (and should always be shortened to Platform when shortened).